DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: deparse
 Title: What the Package Does (one line, title case)
 Version: 0.0-0
-Authors@R: person("Kirill", "Müller", role = c("aut", "cre"), email = "krlmlr+r@mailbox.org")
+Authors@R: person("Kirill", "Muller", role = c("aut", "cre"), email = "krlmlr+r@mailbox.org")
```

Comment:
It threw invalid multibyte character errors on both travis and appveyor. Not sure how to fix that, so it seemed easiest to replace the ü with a u for now.